docs/data-modeling/editing.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Editable table data
+summary: Admins can create, update, and delete data in tables right in Metabase.
+---
+
+# Editable table data
+
+{% plans-blockquote.html feature="Editable table data"  %}
+
+Admins can enable editable table data in Metabase.
+
+![Edit table](./images/edit-table.png)
+
+## Setting up editable table data
+
+_Currently, editable table data is only available for PostgreSQL and MySQL._
+
+To set up table editing, admins can:
+
+1. Click the **gear** icon in the top right.
+2. Select **Admin settings**.
+3. Click **Databases**.
+4. Select your database.
+5. Toggle on **Editable table data**.
+
+For this to work, Metabase's connection to your database must support writing back to its tables. See [privileges to enable actions and editable table data](../databases/users-roles-privileges.md#privileges-to-enable-actions-and-editable-table-data).
+
+## Update or delete records in a table
+
+To update or delete records in a table:
+
+1. Navigate to the left sidebar.
+2. In the Data section, click **Databases**.
+3. Hover over the table you want to edit and click the **Pencil** icon.
+4. Edit rows in the table just like you would in a spreadsheet.
+
+Depending on the data in the column, Metabase may show a list of values to choose from. If you need to add an option to that list, simply type out the option and add it.
```

Comment:
I tried that, and while the record itself was updated with the new value, that value did not show up in the list when editing other values later
<img width="696" height="1166" alt="image" src="https://github.com/user-attachments/assets/b8d85d07-e991-47b0-be64-801b255bc6c4" />